meson.build

```diff
@@ -529,14 +530,22 @@ if have_gumjs
     quickjs_dep_native = dependency('quickjs', native: true)
   endif
 
-  v8_dep = dependency('v8-10.0',
-    version: '>=10.6.122',
+  # https://github.com/frida/v8 exports v8-10.0
+  # vanilla nodejs exports v8
+  v8_dep = dependency(['v8-10.0', 'v8'],
+    version: ['>=10.6.122', '<11'],
     required: get_option('v8'),
+    not_found_message: 'please use v8 from https://github.com/frida/v8'
   )
-  cdata.set('HAVE_V8', v8_dep.found())
   if v8_dep.found()
+    assert(
+      cppc.has_header_symbol('v8-platform.h', 'v8::ThreadingBackend', dependencies: [v8_dep]),
+      'please use v8 from https://github.com/frida/v8'
+    )
+    # TODO use the actual name of the found v8_dep
     gumjs_extra_requires += 'v8-10.0'
```

Comment:
currently frida-gum does not work with vanilla v8, so this can be ignored

Reply:
Right! I'm not sure there's much point in looking for `v8` though. But the `not_found_message` change alone does seem like a good idea.

Reply:
fixed + check for v8::ThreadingBackend